azure/azure.go

```diff
@@ -206,9 +207,16 @@ func NewEnvClient() (cloudops.Ops, error) {
 
 // NewClient creates new client from specified config.
 func NewClient(config Config) (cloudops.Ops, error) {
-	authorizer, err := auth.NewAuthorizerFromEnvironment()
-	if err != nil {
-		return nil, err
+	authorizer := config.Authorizer
```

Comment:
Can we create the authorizer here in the cloudsops layer itself, rather than asking the clients of cloudops to create an authorizer? In this way any user of cloudops (potentially operator) can leverage this new authorizer

Reply:
OK -- this would also cut down on imported packages required by the caller

Reply:
@adityadani -- your feedback addressed in updated [ed8c357](https://github.com/libopenstorage/cloudops/pull/101/commits/ed8c357446b7c95c6584bb260b1f5438a8f364ee)
* can pass string-values via config, to create a different Authorizer